prog/vm/github_runner.rb

```diff
@@ -234,60 +234,12 @@ def setup_info
 
   label def setup_forked_runner
     command = <<~COMMAND
-      curl --output actions-runner.tar.gz -L #{Config.github_cache_forked_runner_tarball_uri}
-
-      rm -rf actions-runner
-      mkdir -p actions-runner
-      tar xzf actions-runner.tar.gz -C actions-runner
-
-      sudo chown -R runneradmin:runneradmin ./actions-runner
-      ./actions-runner/env.sh
-
-      cat <<EOT > ./actions-runner/run-withenv.sh
-      #!/bin/bash
-      mapfile -t env </etc/environment
-      exec env -- "\\${env[@]}" ./actions-runner/run.sh --jitconfig "\\$1"
-      EOT
-      chmod +x ./actions-runner/run-withenv.sh
-
-      jq '. += [#{setup_info.to_json}]' /imagegeneration/imagedata.json > ./actions-runner/.setup_info
-
-      echo "PATH=$PATH" >> ./actions-runner/.env
-
-      sudo rm -rf /home/runner/actions-runner
-      sudo mv ./actions-runner /home/runner/
-      sudo chown -R runner:runner /home/runner/actions-runner
-
       echo "CUSTOM_ACTIONS_CACHE_URL=http://localhost:51123/random_token/" | sudo tee -a /etc/environment
```

Comment:
Since the remaining part is too small, IMHO, you can append it to `command` in `setup_environment`. Something like that

```ruby
    if github_runner.installation.project.get_ff_transparent_cache
      command += <<~COMMAND
        echo "CUSTOM_ACTIONS_CACHE_URL=http://localhost:51123/random_token/" | sudo tee -a /etc/environment
        echo "127.0.0.1 localhost.blob.core.windows.net" | sudo tee -a /etc/hosts
      COMMAND
    end

```